lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -197,6 +216,14 @@ function distributeTwoNodes(firstNode, secondNode) {
   return Node.Creator.parenthesis(Node.Creator.operator('+', newArgs));
 }
 
+function hasFraction(args) {
+  return args.filter(isFraction).length > 0;
+}
+
+function isFraction(node) {
+  return Node.Type.isOperator(node) && node.op === '/';
```

Comment:
you can just use the function `Node.Type.isOperator(node, '/')` - I find that a little cleaner :)